hazelcast/src/main/java/com/hazelcast/map/impl/MerkleTreeNodeEntries.java

```diff
@@ -37,12 +36,12 @@
  */
 public class MerkleTreeNodeEntries implements IdentifiedDataSerializable {
```

Comment:
can we add a note to this class javaDoc to express it is only used for WAN?

Reply:
Yes, we can but I actually purposefully avoided the use of "WAN" when mentioning merkle trees since we might actually use it for intra-cluster replication. WDYT?